pkg/assets/builder.go

```diff
@@ -177,6 +177,7 @@ func (a *AssetBuilder) RemapImage(image string) (string, error) {
 		} else {
 			normalized = strings.TrimPrefix(normalized, "k8s.gcr.io/")
 		}
+		normalized = strings.TrimPrefix(normalized, registryMirror+"/")
```

Comment:
nit: I would add a comment above this to clarify what you are removing and where it could come from. Otherwise it could cost new contributors quite some time to make sense out of this line.

Especially as it's directly "bound" to the default registry removal - which makes it look directly related.

Reply:
Good point.
Added a comment and reworked the code to be easier to understand.